vowpalwabbit/cbify.h

```diff
@@ -4,6 +4,25 @@
 
 #pragma once
 #include "reductions_fwd.h"
+#include "example.h"
+
+template <typename T>
+void v_move(v_array<T>& dst, v_array<T>& src)
```

Comment:
this is just doing a copy now - it should be removed and either copy or move constructor used

Reply:
done